exercises/async-activity-completion/practice/src/main/java/asyncactivitycompletion/VerifyAndCompleteTranslation.java

```diff
@@ -22,13 +22,11 @@ public static void main(String[] args) throws ExecutionException, InterruptedExc
     
     
     // TODO: PART C: Read in the taskToken from args[0] and decode it from Base64
-    byte[] taskToken = Base64.getDecoder().decode(args[0]);
```

Comment:
This is supposed to be removed. This is the answer. It was accidentally left there.